docs/build.md

```diff
@@ -19,7 +19,8 @@ spec:
     image: gcr.io/paketo-buildpacks/builder:base
     imagePullSecrets: 
     - name: builder-secret  
-  cacheName: persisent-volume-claim-name
+  cache:
+    volumeName: persisent-volume-claim-name
```

Comment:
I think `pvcName` is more accurate here

Reply:
I wonder if we should match the long name in [VolumeSource](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#persistentvolumeclaimvolumesource-v1-core) and use the long name `persistentVolumeClaimName`?